ros_bt_py/ros_bt_py/nodes/random_number.py

```diff
@@ -53,7 +53,7 @@ def _do_setup(self):
     def _do_tick(self):
         validate_range(self.options["min"], self.options["max"])
         self.outputs["random_number"] = random.randrange(
-            self.options["min"], self.options["max"]
+            self.options["min"], self.options["max"] + 1
```

Comment:
That sounds good, but we should also adjust the node's documentation.

Reply:
Done